crates/runestick/src/value.rs

```diff
@@ -324,6 +324,12 @@ impl Value {
                 let mut buffer = ryu::Buffer::new();
                 s.push_str(buffer.format(*float));
             }
+            Value::Bool(bool) => {
+                s.push_str(&format!("{}", bool));
```

Comment:
Note that `format!` heap allocates a `String`. This can be avoided by doing something like this instead:
```rust
return Ok(write!(s, "{}", bool));
```
Same below w/ `Byte`.